cmd/gapit/devices.go

```diff
@@ -49,25 +55,29 @@ func (verb *devicesVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 		return log.Err(ctx, err, "Failed to get device list")
 	}
 
-	stdout := os.Stdout
-	for i, p := range devices {
-		fmt.Fprintf(stdout, "-- Device %v: %v --\n", i, p.ID.ID())
+	deviceObjs := []deviceObj{}
+	for _, p := range devices {
 		o, err := client.Get(ctx, p.Path(), nil)
 		if err != nil {
-			fmt.Fprintf(stdout, "%v\n", log.Err(ctx, err, "Couldn't resolve device"))
+			log.Err(ctx, err, "Couldn't resolve device")
```

Comment:
This does not do what you think it does. `log.Err` creates a new error, which you are now ignoring.

Reply:
Thanks for this, fix is here: https://github.com/google/gapid/pull/3737